qutebrowser/browser/commands.py

```diff
@@ -451,9 +451,10 @@ def tab_take(self, index, keep=False):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('win_id', completion=miscmodels.window)
+    @cmdutils.argument('private', flag='p')
```

Comment:
No need for the decorator, as the `-p` flag is already the default for `--private` (the first letter is taken by default)